validation/shapes/ontologyShapes.ttl

```diff
@@ -0,0 +1,146 @@
+# imports: http://datashapes.org/dash
+# imports: http://topbraid.org/tosh
+# imports: http://www.w3.org/ns/shacl#
+
+@prefix dash: <http://datashapes.org/dash#> .
+@prefix gist: <https://w3id.org/semanticarts/ns/ontology/gist/> .
+@prefix gshapes: <https://shapes.semanticarts.com/gist/> .
```

Comment:
Does anyone have a non-gist namespace suggestion for the shape instances? We could remove the `/gist/` at the end.